0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -288,16 +303,15 @@ means that a transfer you created has timed out.
 <code>class Transfer</code>
 
 The `Transfer` class is used to describe local ledger transfers. Only
-[id](#id), [account](#account), [ledger](#ledger), and [amount](#amount) are required; the other
+[id](#id), [address](#address), [ledger](#ledger), and [amount](#amount) are required; the other
 fields can be left undefined (but not any other false-y value) if unused.
 
 ###### Fields
 | Type | Name | Description |
 |:--|:--|:--|
 | `String` | [id](#id) | UUID used as an external identifier |
-| `String` | [account](#account) | Local source or destination account ID |
+| `String` | [address](#address) | ILP Address of the source or destination account |
 | `String` | [amount](#amount) | Decimal transfer amount |
-| `String` | [ledger](#ledger) | Ledger address |
```

Comment:
I think `ledger` would be there and be the prefix, e.g. `example.ledger.` - it's convenient to have so you can easily tell which ledger plugin this came from / should go to.